prusti-contracts/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![no_std]
+#![feature(decl_macro)]
```

Comment:
Why is this needed? I do not see any macro declaration.
```suggestion
#![feature(decl_macro)]
```